server/src/sim/financialanalysis.py

```diff
@@ -48,7 +47,7 @@ def financialanalysis(D, postyear=2015, S=None, rerunmodel=False, artgrowthrate=
 
     # Get most recent ART unit costs 
     progname = 'ART'
-    prognumber = D['data']['meta']['progs']['short'].index(progname)
```

Comment:
Besides, with this change all "old" projects uploaded from json will not work, because they have D['programs'] as a dictionary already.